index.html

```diff
@@ -4,242 +4,245 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Sam's Personal Portfolio</title>
-    <link rel="stylesheet" href="styles.css">
-    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Poppins" />
+    <title>Sam's Portfolio</title>
+    <script src="https://kit.fontawesome.com/1e6c06b0c6.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="desktop.css">
+    
 </head>
 <body>
-    <div class="page1">
-    <header class="header">
-        <nav class="nav">
-            <div class="myLogo">
-               <a href="#">My Logo</a> 
+    <div class="container">
+        <header class="header">
+            <div class="logo">
+                <a href="#">My Logo</a>
```

Comment:
- [ ] Your header elements are not aligned properly, I checked the CSS of the header class and you are using `align-content` instead of `align-items`. The `align-content` property has a different context and it won't work in this case, also please make sure that each link color matches the design.
![image](https://user-images.githubusercontent.com/33767951/139553793-ceed07a3-5904-4f12-9ccf-a4c05b8f399b.png)